packages/experimental/Avatar/FluentUIReactNativeAvatar.podspec

```diff
@@ -14,13 +14,13 @@ Pod::Spec.new do |s|
   s.source           = { :git => "https://github.com/microsoft/fluentui-react-native.git", :tag => "#{s.version}" }
   s.swift_version    = "5"
 
-  s.ios.deployment_target = "11.0"
+  s.ios.deployment_target = "13.0"
   s.ios.source_files      = "ios/*.{swift,h,m}"
-  s.ios.dependency 'MicrosoftFluentUI/Controls_ios', '~> 0.1.28'
+  s.ios.dependency 'MicrosoftFluentUI', '~> 0.2.2'
 
   s.osx.deployment_target = "10.14"
   s.osx.source_files      = "macos/*.{swift,h,m}"
-  s.osx.dependency 'MicrosoftFluentUI/AvatarView_mac', '~> 0.1.28'
+  s.osx.dependency 'MicrosoftFluentUI', '~> 0.2.2'
```

Comment:
FYI, I had some trouble with using the subspecs and had to use the full library. I'll try again later..

Reply:
hmmm i am curious what is the issue is.

Reply:
Looks like Mac controls work, but when I try to set iOS, I get this:

`  s.ios.dependency 'MicrosoftFluentUI/Avatar_ios', '~> 0.2.2'`
```
[!] CocoaPods could not find compatible versions for pod "MicrosoftFluentUI/Avatar_ios":
  In Podfile:
    FluentUI-React-Native-Avatar (from `../../../packages/experimental/Avatar/FluentUIReactNativeAvatar.podspec`) was resolved to 0.5.0, which depends on
      MicrosoftFluentUI/Avatar_ios (~> 0.2.2)

None of your spec sources contain a spec satisfying the dependency: `MicrosoftFluentUI/Avatar_ios (~> 0.2.2)`.

You have either:
 * mistyped the name or version.
 * not added the source repo that hosts the Podspec to your Podfile.
 ```
 
 It tells me it's either not published, or something else weird is happening. Either way, I think using the full pod spec here doesn't affect binary size as the FURN static libraries only contain the FURN symbols.

Reply:
The subspecs definitions changed after 0.2.2 was published. These changes created the **Avatar_ios** subspec you're trying to use.
I expect that 0.2.3 will work just fine with the changes consuming that subspec only.

Reply:
Cool! I'll follow up and see if this works after 0.2.3 is out =)